Chart.yaml

```diff
@@ -2,10 +2,10 @@ apiVersion: v2
 name: go-ratelimit
 description: A Ratelimit Service Helm chart for Kubernetes
 type: application
-version: 1.0.2
+version: 1.0.3
 appVersion: 1.5.0
```

Comment:
Hey @santinoncs 
Linking to my [comment](https://github.com/softonic/go-ratelimit-chart/issues/6#issuecomment-845848638), it seems that version `1.5.0` is not available as a docker image. So maybe it makes sense to change the default image tag in the values file.